js/__internal/grids/grid_core/editing/m_editing_cell_based.ts

```diff
@@ -387,10 +391,8 @@ const editingControllerExtender = (Base: ModuleType<EditingController>) => class
       const columnsCount = this._columnsController.getVisibleColumns().length;
       changes.forEach(({ key }) => {
         const rowIndex = this._dataController.getRowIndexByKey(key);
-        if (rowIndex !== -1) {
-          for (let columnIndex = 0; columnIndex < columnsCount; columnIndex++) {
-            this._rowsView._getCellElement(rowIndex, columnIndex).removeClass(CELL_MODIFIED_CLASS);
-          }
+        for (let columnIndex = 0; columnIndex < columnsCount; columnIndex++) {
+          this._rowsView._getCellElement(rowIndex, columnIndex)?.removeClass(CELL_MODIFIED_CLASS);
```

Comment:
Minor comment

Let's move ```cellElement``` to a separate variable:

```suggestion
          const cellElement = this._rowsView._getCellElement(rowIndex, columnIndex);
          cellElement?.removeClass(CELL_MODIFIED_CLASS);
```